docs/integrations/pydantic.md

```diff
@@ -67,6 +67,48 @@ class UserInput:
     pass
 ```
 
+## Interface types
+
+Interface types are similar to normal types; we can create one by using the
+`strawberry.experimental.pydantic.interface` decorator:
+
+```python
+import strawberry
+from pydantic import BaseModel
+from typing import List
+
+# pydantic types
+class User(BaseModel):
+    id: int
+    name: str
+
+class NormalUser(User):
+    friends: List[int] = []
+
+class AdminUser(User):
+    role: int
+
+# strawberry types
+@strawberry.experimental.pydantic.interface(model=User, fields=[
+    'id',
+    'name',
+])
+class UserType:
+    pass
+
+@strawberry.experimental.pydantic.type(model=NormalUser, fields=[
+    'friends',
+])
+class NormalUserType(UserType):  # note the base class
```

Comment:
do you think we should automatically detect interfaces?

Reply:
I'm not sure how you would do that, as the base class in pydantic isn't really distinguishable from a normal pydantic class. It just has subclasses that are relevant, but we don't know that when we're inspecting the base class.

Reply:
gotcha :)